### PR TITLE
[IMP] various: batch of small improvements

### PR DESCRIPTION
**Pull request by jeh-odoo**
Batch of small improvements like labels, design, ...
See commits for more details.

task-3515819

### CODE REVIEW

File: addons/mass_mailing_sms/views/mailing_mailing_views.xml [hunk 1]
```diff
@@ -27,12 +27,12 @@
...
                 <button name="action_put_in_queue" type="object"
                     string="Send" class="oe_highlight" data-hotkey="v"
-                    invisible="mailing_type == 'mail' or state in ('in_queue', 'sending', 'processing', 'done') or sms_force_send"
-                    confirm="This will send SMS to all recipients. Do you still want to proceed?"/>
+                    invisible="mailing_type == 'mail' or state in ('in_queue', 'sending', 'processing', 'done') or sms_force_send"/>
                 <button name="action_send_mail" type="object"
                     string="Send Now" class="oe_highlight" data-hotkey="g"
                     invisible="mailing_type == 'mail' or state in ('in_queue', 'processing', 'done') or schedule_type == 'scheduled' or not sms_force_send"
-                    confirm="This will send SMS to all recipients now. Do you still want to proceed?"/>
+                    confirm="Once you send these text messages, they'll be making a grand entrance in all pockets, creating quite the bzzzzz!"
```
Comment (tde-banana-odoo) [inline addons/mass_mailing_sms/views/mailing_mailing_views.xml:34]: :disguised_face:

**Review by std-odoo — COMMENTED** [review]
Hi :) 
For the truck image, shouldn't we change the CSS of the kanban ?
Otherwise LGTM

**Review by Whenrow — APPROVED** [review]

**Comment by kmagusiak** [comment]
Still valid?

**Comment by jeh-odoo** [comment]
Was handled by https://github.com/odoo/odoo/pull/148850

**Pull request closed without merging**